tests/feature_lists/test_wikidatawiki.py

```diff
@@ -93,3 +93,11 @@ def test_is_astronomical_object(q7251):
                  cache={entity: crab_nebula}) is True
     assert solve(wikidatawiki.is_astronomical_object,
                  cache={entity: q7251}) is False
+
+def test_has_image(q7251):
```

Comment:
I'd say test for item that doesn't have image either, you can turn the crab nebula I put above into a fixture and use that instead.

Reply:
Done ✅